CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake;${CMAKE_MODULE_PATH}")
 include(Utilities)
 project(KinesisVideoProducerC VERSION 1.5.1 LANGUAGES C)
 
+set(KINESIS_VIDEO_PRODUCER_C_MAJOR_VERSION 1)
```

Comment:
Can we update this section to be modified to upcoming version? And the `project` line right above this too.

Reply:
Done